packages/expo-dev-menu/app/components/Views.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { useTheme } from '@react-navigation/native';
+import { View, ScrollView } from 'react-native';
+
+import Colors from '../constants/Colors';
+
+type ViewProps = View['props'];
```

Comment:
Aren't there `ViewProps` already defined in `react-native`? 🤔 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts